Snippets/Extensions.Logging/Extensions.Logging_2/Extensions.Logging_2.csproj

```diff
@@ -4,7 +4,7 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="NServiceBus.Extensions.Logging" Version="2.0.0-beta.1" />
+    <PackageReference Include="NServiceBus.Extensions.Hosting" Version="2.0.0-rc.1" />
```

Comment:
Why was the package changed? Are we obsoleting it? If so, then we cannot use this sample?

Reply:
Those were incorrect to change. I have updated them since (as this was still a work-in-progress).